manifests/charts/ztunnel/values.yaml

```diff
@@ -3,6 +3,7 @@ hub: gcr.io/istio-testing
 # Tag to pull from. Image will be `Hub/Image:Tag-Variant`
 tag: latest
 # Variant to pull. Options are "debug" or "distroless". Unset will use the default for the given version.
+# Setting variant to "debug" will also run with higher permissions to allow tcpdump.
```

Comment:
`kubectl debug` may be more appropriate here?

This is going to be a privilege escalation for users by default which doesn't seem great. `debug` is really just `ubuntu`, we may even only allow distroless as part of "ambient uses safe mode by default".

Reply:
Would kubectl debug raise the capabilities of the pod ? I.e. if a cap is dropped, will it be added back ? 

I tried and didn't seem to work - but if there is some option to add the caps I agree it would be far better.

Reply:
Also "safe by default" is all good - but "debug" is neither the default nor supposed to be safe.

Reply:
debug _is_ the default. It shouldn't be in safe mode, of course.

I just tested and it worked. `kubectl debug --image istio/base -it  -n istio-system ztunnel-9xpvt` then `tcpdump` in the shell had no issues. `kubectl exec` into ztunnel I got permission denied

Reply:
Ok, will remove this. 

Speaking of safe mode - for ztunnel it may be good to default to safe, and have debug optional. We should do
the same in the safe charts for istiod.

Reply:
Also - if this works, maybe we should remove tcpdump, dig and other debug helpers from istio/base ? 
I guess backward compat will prevent this - but we can create a second base for 'safe mode' at some point 
and do a base cleanup too.